go.mod

```diff
@@ -6,9 +6,8 @@ require (
 	github.com/ardielle/ardielle-go v1.5.2
 	github.com/stretchr/testify v1.3.0
 	github.com/yahoo/athenz v1.8.24
-	github.com/yahoo/k8s-athenz-istio-auth v0.0.0-20190717030003-19444f7f59a2
-	k8s.io/api v0.0.0-20190222213804-5cb15d344471
-	k8s.io/apimachinery v0.0.0-20190629003722-e20a3a656cff
+	github.com/yahoo/k8s-athenz-syncer v0.1.1
+	k8s.io/api v0.0.0-20190817221950-ebce17126a01
+	k8s.io/apimachinery v0.0.0-20190817221809-bf4de9df677c
 	k8s.io/client-go v10.0.0+incompatible
```

Comment:
can we update client-go to v12 (compat with 1.15)?

Reply:
updated